src/screens/Library.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import ButtonComponent from "../components/Button";
+import Loader from "../components/Loader";
+import Card from "../components/Card";
+import { useFetch, API_Candidates } from "../utility";
+import Likes from "./Likes";
+import styles from "./Profile.module.css";
+import Tags from "../components/Tags";
+
+const Library = () => {
+  const [dataCandidate, loadingCandidate] = useFetch(API_Candidates);
+  const [display, setDisplay] = useState(true);
+
+  return (
+    <>
+      <div onClick={() => setDisplay(!display)}>
```

Comment:
you can remove the wrapping div and pass the function you want to execute as a prop to ButtonComponent